src/utils/SongUtils.tsx

```diff
@@ -13,6 +13,8 @@ export const shlJsonUrl =
 
 export const SHL_BOOK_ID = "shl";
 
+const emptyMap = new Map();
```

Comment:
nit: If this isn't being used anywhere else, let's just inline it in `PLACEHOLDER_SONG`

Reply:
Done 👍